### ISSUE TITLE
Exception loading blockstate definitions.

### ISSUE THREAD

**Issue by K0-RR** [Qelifern/IronFurnaces#25]
<details>
  <summary>Crashlog</summary>

```
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/iron_furnace.json' missing model for variant: 'ironfurnaces:iron_furnace#lit=false,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/iron_furnace.json' missing model for variant: 'ironfurnaces:iron_furnace#lit=true,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/iron_furnace.json' missing model for variant: 'ironfurnaces:iron_furnace#lit=false,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/iron_furnace.json' missing model for variant: 'ironfurnaces:iron_furnace#lit=true,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/gold_furnace.json' missing model for variant: 'ironfurnaces:gold_furnace#lit=false,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/gold_furnace.json' missing model for variant: 'ironfurnaces:gold_furnace#lit=true,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/gold_furnace.json' missing model for variant: 'ironfurnaces:gold_furnace#lit=false,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/gold_furnace.json' missing model for variant: 'ironfurnaces:gold_furnace#lit=true,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/diamond_furnace.json' missing model for variant: 'ironfurnaces:diamond_furnace#lit=true,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/diamond_furnace.json' missing model for variant: 'ironfurnaces:diamond_furnace#lit=true,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/diamond_furnace.json' missing model for variant: 'ironfurnaces:diamond_furnace#lit=false,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/diamond_furnace.json' missing model for variant: 'ironfurnaces:diamond_furnace#lit=false,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/emerald_furnace.json' missing model for variant: 'ironfurnaces:emerald_furnace#lit=true,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/emerald_furnace.json' missing model for variant: 'ironfurnaces:emerald_furnace#lit=false,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/emerald_furnace.json' missing model for variant: 'ironfurnaces:emerald_furnace#lit=false,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/emerald_furnace.json' missing model for variant: 'ironfurnaces:emerald_furnace#lit=true,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/crystal_furnace.json' missing model for variant: 'ironfurnaces:crystal_furnace#lit=false,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/crystal_furnace.json' missing model for variant: 'ironfurnaces:crystal_furnace#lit=true,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/crystal_furnace.json' missing model for variant: 'ironfurnaces:crystal_furnace#lit=true,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/crystal_furnace.json' missing model for variant: 'ironfurnaces:crystal_furnace#lit=false,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/obsidian_furnace.json' missing model for variant: 'ironfurnaces:obsidian_furnace#lit=false,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/obsidian_furnace.json' missing model for variant: 'ironfurnaces:obsidian_furnace#lit=true,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/obsidian_furnace.json' missing model for variant: 'ironfurnaces:obsidian_furnace#lit=true,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/obsidian_furnace.json' missing model for variant: 'ironfurnaces:obsidian_furnace#lit=false,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/netherite_furnace.json' missing model for variant: 'ironfurnaces:netherite_furnace#lit=false,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/netherite_furnace.json' missing model for variant: 'ironfurnaces:netherite_furnace#lit=true,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/netherite_furnace.json' missing model for variant: 'ironfurnaces:netherite_furnace#lit=false,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/netherite_furnace.json' missing model for variant: 'ironfurnaces:netherite_furnace#lit=true,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/copper_furnace.json' missing model for variant: 'ironfurnaces:copper_furnace#lit=true,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/copper_furnace.json' missing model for variant: 'ironfurnaces:copper_furnace#lit=true,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/copper_furnace.json' missing model for variant: 'ironfurnaces:copper_furnace#lit=false,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/copper_furnace.json' missing model for variant: 'ironfurnaces:copper_furnace#lit=false,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/silver_furnace.json' missing model for variant: 'ironfurnaces:silver_furnace#lit=true,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/silver_furnace.json' missing model for variant: 'ironfurnaces:silver_furnace#lit=true,facing=down'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/silver_furnace.json' missing model for variant: 'ironfurnaces:silver_furnace#lit=false,facing=up'
[11:02:35] [Worker-Main-8/WARN]: Exception loading blockstate definition: 'ironfurnaces:blockstates/silver_furnace.json' missing model for variant: 'ironfurnaces:silver_furnace#lit=false,facing=down'
```

**Comment by XenoMustache** [comment]
Can you provide more information? Is this causing a crash? What version are you using? Why didn't you use the issue template properly?

**Comment by K0-RR** [comment]
Fabric 1.16.4; It's not crashing, just printing this in the console; v. 1.16.2-2.2.1; I didn't use a template because I thought You would know about this type of issue and I was lazy sorry for this.

**Comment by K0-RR** [comment]
I'm reporting this type of stuff (it's not only in your mod, a few others too) to make mods spam less the logs.

**Comment by Qelifern** [comment]
This issue is fixed in 2.3.0 Fabric, Properties.FACING is including UP and DOWN so the Properties.FACING I shouldve been using is Properties.HORIZONTAL_FACING, which only includes horizontal facings (no shit sherlock) which in turn fixes this exception.
Also the Fabric version of the mod has a separate github page now: https://github.com/Qelifern/FabricIronFurnaces/issues
Thank you.
